server/issue.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/v28/github"
```

Comment:
this is to lock the version?

Reply:
Kind of. This is just how imports work with go modules. The version _has_ to be part of the module path. See https://github.com/golang/go/wiki/Modules#semantic-import-versioning and https://github.com/google/go-github/blob/master/go.mod#L1

Reply:
👍